client/views/meet/MeetPage.js

```diff
@@ -14,20 +16,36 @@ function MeetPage() {
 	const roomId = useRouteParameter('rid');
 	const visitorToken = useQueryStringParameter('token');
 	const layout = useQueryStringParameter('layout');
+	const [visitorName, setVisitorName] = useState('');
+	const [agentName, setAgentName] = useState('');
+	const [isAgentRemote, setIsAgentRemote] = useState('false');
```

Comment:
Based on my understanding these 2 variables i.e. `isAgentRemote` and `isAgentLocal` seems to be redundant. I think you're using this to check if the user is a visitor or an agent. If so, then I'd recommend either
1. using `visitorToken` token param (it will be set for visitor and empty for agent) (recommended IMO) or 
2. Define only one variable like `isAnAgent`